### PR TITLE
Update test_export_login.py

### PR DESCRIPTION
**Pull request by heesen3**
- [x] Have you followed the [guidelines for contributing](https://github.com/snapcore/snapcraft/blob/master/CONTRIBUTING.md)?
- [ ] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?
- [ ] If this is a bugfix. Have you checked that there is a bug report open for the issue you are trying to fix on [bug reports](https://bugs.launchpad.net/snapcraft)?
- [ ] If this is a new feature. Have you discussed the design on the [forum](https://forum.snapcraft.io)?
- [x] Have you successfully run `./runtests.sh static`?
- [x] Have you successfully run `./runtests.sh unit`?

-----

### CODE REVIEW

File: snapcraft/tests/unit/commands/test_export_login.py [hunk 1]
```diff
@@ -1,7 +1,4 @@
-# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
```
Comment (elopio) [inline snapcraft/tests/unit/commands/test_export_login.py]: Why did you remove this line?

File: snapcraft/tests/unit/commands/test_export_login.py [hunk 2]
```diff
@@ -132,3 +129,50 @@ def test_failed_login_with_invalid_credentials(self, mock_login):
         self.assertThat(result.output, Contains(
             storeapi.constants.INVALID_CREDENTIALS))
         self.assertThat(result.output, Contains('Login failed.'))
+    store = storeapi.StoreClient()
+    snap_name = store.get_snap_name_for_id('edge123')
```
Comment (elopio) [inline snapcraft/tests/unit/commands/test_export_login.py]: This is weird, the indentation is not right. Can you explain what are you trying to do here? We might be able to guide you better if you document your intention and reasons on the pull request description.

**Comment by heesen3** [comment]
I was told by Kyrofa that the file store.py does not have enough coverage line 65 and 67.
He told me I can get coverage for these by adding a test to test that the parameter --snap is working when the command snapcraft export_login__help is running in terminal.
I did as I was told but  i am getting the error snap is not found in 16 series i registered he snap in the store but i am still getting he same error.

**Comment by elopio** [comment]
OK @heesen3. But you still have basic indentation problems on your branch.

Can you please follow http://www.diveintopython3.net/ ? at least until the unittests section. That way you will understand python better. If you have any questions while you follow the text, let us know in IRC.

File: snapcraft/tests/unit/commands/test_export_login.py (hunk 1 above)
Comment (heesen3) [reply snapcraft/tests/unit/commands/test_export_login.py]: it was a mistake i have put it back

Comment (elopio) [reply snapcraft/tests/unit/commands/test_export_login.py]: You still have an extra space in there.

File: snapcraft/tests/unit/commands/test_export_login.py [hunk 3]
```diff
@@ -95,31 +95,31 @@ def test_successful_login_with_2fa(
...
+                mock.call('Email: '), mock.call('Second-factor auth: ')])
         self.assertThat(mock_login.call_count, Equals(2))
         mock_login.assert_has_calls([
-            mock.call(
-                'user@example.com', mock.ANY, acls=None, packages=None,
-                channels=None, save=False, config_fd=None),
-            mock.call(
-                'user@example.com', mock.ANY, one_time_password='123456',
-                acls=None, packages=None, channels=None, save=False,
-                config_fd=None)])
```
[the remark this answers is not in the document]
Comment (heesen3) [reply snapcraft/tests/unit/commands/test_export_login.py]: ok will do

File: snapcraft/tests/unit/commands/test_export_login.py [hunk 4]
```diff
@@ -130,5 +130,41 @@ def test_failed_login_with_invalid_credentials(self, mock_login):
...
+    def test_export_with_snap_name(
+            self, mock_acl, mock_login, mock_get_account_information):
+        self.mock_input.return_value = 'user@example.com'
+        mock_acl.return_value = {
+            'snap_ids': None,
+            'channels': ['edge'],
+            'permissions': None,
+        }
+
+        result = self.run_command(['export-login', 'exported'])
```
Comment (elopio) [inline snapcraft/tests/unit/commands/test_export_login.py]: you are not passing the --snaps , so this test doesn't have anything different from test_successful_export, right?

Comment (heesen3) [reply snapcraft/tests/unit/commands/test_export_login.py]: when i am trying to pass --snap i get the error Snap 'myapp' was not found in '16' series.

Comment (heesen3) [reply snapcraft/tests/unit/commands/test_export_login.py]: what should i do ?

Comment (kyrofa) [reply snapcraft/tests/unit/commands/test_export_login.py]: @elopio would you prefer a mock, or an update to the fake store?

**Comment by elopio** [comment]
@heesen3 on this last iteration you changed the name of the file where the exported login will be saved. That's not what this test requires. Please push the one that gives you the error about the snap not being in the store, so we can take a look at what you are missing there.

**Comment by heesen3** [comment]
ok i will send the one with the error

**Comment by heesen3** [comment]
https://pastebin.com/M8FaHraA

**Comment by kyrofa** [comment]
Closing due to inactivity. Please re-propose once you've addressed the issues.
